sharing/index.md

```diff
@@ -0,0 +1,47 @@
+---
+tags:
+  - sharing
+author:
+  display_name: AGRC Office
+  email: agrc@utah.gov
+date: 2020-01-13 17:00:00
+title: Sharing GIS Data
+categories: []
+---
+
+#### Quick Links
+
+[Submitting Data to the SGID]({% link sharing/process/index.md %}) — [Basic Metadata Creation Process]({% link sharing/metadata/index.md %}) — [AGRC Sharing Policies]({% link about/policy/index.md %})
+
+## The Joys of Sharing
```

Comment:
```suggestion
## The SGID
```

Reply:
I'll address the heading names as part of the track-based rewrite.